File: ClientApp/src/app/models/common/emailDirectory.ts
export class EmailDirectory{
  id: string;
  name: string;
  designation: string;
  email: string;
  phoneNumber: string;
}

export class EmailDirectoryViewModel {
  constructor(directoryListModel?: EmailDirectory[], totalCount?: number) {
    this.directoryListModel = new Array<EmailDirectory>();
    this.totalCount = totalCount;

  }
  public totalCount: number;
  public directoryListModel: EmailDirectory[];

}

File: A4.Empower-Git/A4.Empower/DAL/Entites/Recruitment/EmailDirectory.cs
using A4.DAL.Entites;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace A4.DAL.Entites
{
    public class EmailDirectory: AuditableEntity
    {
        #region Constructor

        public EmailDirectory()
        {
            Id = Guid.NewGuid();
        }
        #endregion

        
        #region Properties
        
        public Guid Id { get; set; }

        
        public string Name { get; set; }

        
        public string Designation{ get; set; }

        public string Email { get; set; }

        public string PhoneNumber { get; set; }

        

        #endregion
    }
}
